doc/contributing/releases.md

```diff
@@ -883,6 +883,13 @@ same GPG key!**
 Use `tools/release.sh` to promote and sign the build. Before doing this, you'll
 need to ensure you've loaded the correct ssh key, or you'll see the following:
 
+If you are using the combination of `zsh`, macOS and your GPG key is protected
+by a password, you might need to run:
```

Comment:
```suggestion
If you your GPG key is protected by a password, you might need to run:
```

I use this workaround in Linux too (Ubuntu), I thought that it was a local bug in my case but seems like it is a common thing 😅